whd/dispatcher.py

```diff
@@ -279,18 +298,28 @@ def _set_pr_labels(self, pr_event, resources) -> bool:
             for resource in resources if resource.source.get('label') is not None
         }
         if not required_labels:
-            return False
+            return True
```

Comment:
what's the semantics of the returned boolean value (and why was the previous value wrong)?

Reply:
(already discussed elsewhere)
according to the docstring, this function is to return true if "if the required label was set"